Fix CSS selectors for tab handler

urbantrout opened the pull request.

urbantrout committed:
remove legacy setup for local development

urbantrout committed:
fix tab-handler

urbantrout committed:
Merge branch 'craft-webpack' of https://github.com/nystudio107/craft into craft-webpack

khalwat commented:
I'll roll this into the next release, I just missed it this time through, sorry.

urbantrout committed:
Merge branch 'craft-webpack' of github.com:nystudio107/craft into craft-webpack

urbantrout committed:
fix php container crash on initial docker start

this can only be merged once
https://github.com/craftcms/cms/pull/5811 gets merged

urbantrout committed:
Merge branch 'craft-webpack' of github.com:nystudio107/craft into craft-webpack

khalwat commented:
I'm going to change this manually, because I took too long to merge this PR, and a few things our out of sync.
